test/csn_bridge.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+# Integration test script for the csn and bridge node.
+# Assumes that `the following binaries exist:
+# 	`bitcoind`, `bitcoin-cli, `utreexo`
+
```

Comment:
would be cleaner to use `set -euo pipefail`. see more at https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/